Resources/Prototypes/_NF/Entities/Objects/Tools/weapon_cases.yml

```diff
@@ -72,5 +73,6 @@
       - EnergySword
       - ItemToggleDisarmMalus
       - Blocking
+      - Explosive
```

Comment:
I made weapon cases as a visual aid for players: some weapons are exceedingly hard to spot even without environmental modifiers that decrease visibility. I don't think that cases are used by players to store weapons or ammo, they usually just loot weapons from cases while leaving the cases behind. There is no mechanical benefits in using weapon cases for storage purposes.
What I'm leading with that is maybe this issue isn't a big deal and can be overlooked? Idk honestly.

Reply:
FYI, This fails since fills happen on server side but the fill into storage is starting client side to server side I think.
So the explosives cannot be inserted based on comp acting server side and not shared

TLDR, While not a big error its a bit stinky.

Reply:
A'ight, I've added empty comp. No error messages upon inserting explosives in weapon cases. Also gave some explosion resistance to cases.